onnxruntime/core/platform/posix/env.cc

```diff
@@ -83,6 +84,25 @@ long int TempFailureRetry(TFunc retriable_operation, TFuncArgs&&... args) {
   return result;
 }
 
+int nftw_remove(
```

Comment:
What does nftw mean?

Reply:
see man nftw (e.g., http://man7.org/linux/man-pages/man3/ftw.3.html)

n "file tree walk", not sure about the n :)

Reply:
add a comment to clarify

---
In reply to: [393225728](https://github.com/microsoft/onnxruntime/pull/3174#discussion_r393225728) [](ancestors = 393225728)